pkg/kubectl/describe/versioned/describe.go

```diff
@@ -302,8 +302,15 @@ func printUnstructuredContent(w PrefixWriter, level int, content map[string]inte
 }
 
 func smartLabelFor(field string) string {
-	commonAcronyms := []string{"API", "URL", "UID", "OSB", "GUID"}
+	// skip creating smart label if field name contains
+	// special characters other than '-'
+	if strings.IndexFunc(field, func(r rune) bool {
+		return (r < 'A' || r > 'Z') && (r < 'a' || r > 'z') && r != '-'
```

Comment:
What about [`unicode.IsLetter`](https://golang.org/pkg/unicode/#IsLetter)?

Reply:
Replaced the range check with unicode.IsLetter.